docs/whats-new.asciidoc

```diff
@@ -16,125 +16,120 @@ Other versions: {security-guide-all}/8.12/whats-new.html[8.12] | {security-guide
 The following enhancements have been added to detection rules and alerts:
 
 [float]
-=== Per-field diff for Elastic prebuilt rule updates
+=== Value list improvements
 
-When examining an {security-guide}/prebuilt-rules-management.html#update-prebuilt-rules[updated Elastic prebuilt detection rule], you can now view rule changes field by field as well as in a full JSON view.
+You can now {security-guide}/value-lists-exceptions.html#edit-value-lists[edit value lists] directly from the UI and wherever you use them. For example, you can add items to a value list while creating a rule exception that references that value list.
 
 [role="screenshot"]
-image::whats-new/images/8.13/prebuilt-rules-update-diff.png[Prebuilt rule comparison, 85%]
+image::whats-new/images/8.14/edit-value-lists.png[Edit items in a value list, 90%]
 
 [float]
-=== Alert suppression supported for indicator match rules
+=== ES|QL fields can be added as custom highlighted fields
 
-{security-guide}/alert-suppression.html[Alert suppression] now supports the {security-guide}/rules-ui-create.html#create-indicator-rule[indicator match] rule type. You can use it to reduce the number of repeated or duplicate detection alerts created by an indicator match rule.
+When adding custom highlighted fields to an ES|QL rule, you can now {security-guide}/rules-ui-create.html#custom-highlighted-esql-fields[specify any fields returned by the rule's query]. This allows you to surface ES|QL fields that contain useful information for investigating alerts.
 
 [float]
-=== Refined header design for alert details flyout 
+=== Editable setup guide field for detection rules
 
-The header design for the {security-guide}/view-alert-details.html[alert details flyout] has been refined to improve readability and structure. Basic alert details now appear clearer and more organized. 
+You can now {security-guide}/rules-ui-create.html#rule-ui-advanced-params[edit the **Setup guide** field] for user-created custom rules. Use this informational field to list rule prerequisites such as required integrations, configuration steps, and anything else needed for the rule to work correctly.
 
 [role="screenshot"]
-image::whats-new/images/8.13/alert-details-flyout-right-panel.png[Right panel of the alert details flyout, 75%]
+image::whats-new/images/8.14/setup-guide-field.png[Setup guide field]
 
 [float]
-== Persistence of Data Quality dashboard results 
+=== Alert suppression for custom query rules is now generally available
```

Comment:
Here's a generic title you can use so you can group the alert suppression changes together. Feel free to use something different if you have another title in mind. :)

```suggestion
=== Alert suppression improvements 
```